solution/ui/regulations/eregs-component-lib/src/components/shared-components/Statutes/utils/urlMethods.js

```diff
@@ -0,0 +1,38 @@
+// SSA
+/**
+ * @param {number} title - CFR title (ex: 42)
+ * @param {string} usc - USC section
+ *
+ * @returns {string} url - url to the House.gov page for the USC section
+ */
+const houseGovUrl = ({ title, usc }) =>`https://uscode.house.gov/view.xhtml?hl=false&edition=prelim&req=granuleid%3AUSC-prelim-title${title}-section${usc}`;
+
+/**
+ * @param {string} souce_url - URL containing COMPS number
+ *
+ * @returns {string} url - url to GovInfo.gov page displaying Statute Compilation PDF
+ */
+const statuteCompilationUrl = ({ source_url }) => {
+    const compsNumber = source_url
+        .split("/")
```

Comment:
Can you add a vitest for this utility function?

Reply:
I will in the next story, EREGCSC-1923.  That's the integration/polish story that will ship the feature to users.  I've already started a branch with some updated Cypress tests, but now I'm at the point where this PR needs to be in `main` so I can continue.